### PR TITLE
Added page for NoiPA in italian version

### PR DESCRIPTION
**Pull request by biancini**
Created page for NoiPA after discussion with MEF.
The page has been anticipated to MEF for approval and can be merged.

### CODE REVIEW

File: _platforms/it/noipa.md [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+La creazione di un sistema unico centralizzato di gestione del personale determinerà una riduzione dei costi per 
+l’erogazione del servizio, anche grazie alle economie di scala generate dall’ampliamento del numero delle amministrazioni 
+servite.
+
+NoiPA è sempre più aperto alla collaborazione con chi fa della ricerca e dell’innovazione il suo obiettivo.
+Se sei un ente di ricerca o una università interessata a collaborare con NoiPA per la realizzazione di progetti innovativi
+di livello nazionale e internazionale puoi collaborare con NoiPA.
+
+<a class="btn btn-primary" href="https://noipa.mef.gov.it/"><i class="it-pa" />Visita il portale NoiPA</a>
+<a class="btn btn-primary" href="https://www.cloudifynoipa.it"><i class="it-code-circle" />Conosci il programma Clodify NoiPA</a>
```
Comment (libremente) [inline _platforms/it/noipa.md]:
```suggestion
<a class="btn btn-primary" href="https://www.cloudifynoipa.it"><i class="it-code-circle" />Conosci il programma Cloudify NoiPA</a>
```

File: _platforms/it/noipa.md [hunk 2]
```diff
@@ -0,0 +1,90 @@
...
+    - title: Ontologia di NoiPA
+      url: https://sparql-noipa.mef.gov.it/endpoint/download/owl/NoiPA.owl
+      icon: file
+      desc: File OWL dell'ontologia dei dati di NoiPA.
+    - title: Dataset
+      url: https://dati-noipa.mef.gov.it/cl/web/open-data/dataset
+      icon: file
+      desc: I vari dataset pubblicati sono consultabili alla pagina Dataset.
+    - title: Endpoint
+      url: http://sparql-noipa.mef.gov.it/sparql
```
Comment (libremente) [inline _platforms/it/noipa.md]:
```suggestion
      url: https://sparql-noipa.mef.gov.it/sparql
```
Maybe https?

**Comment by sebbalex** [comment]
English version for this platform makes sense?

**Comment by libremente** [comment]
> English version for this platform makes sense?

It may make sense, yes. However, I'd merge this now and then open a dedicated issue + PR for the translation. @sebbalex

File: _platforms/it/noipa.md [hunk 3]
```diff
@@ -0,0 +1,90 @@
+---
+title: NoiPA
+subtitle: servizi PA a Persone PA
+logo: /assets/images/logo-noipa.png
+payoff: NoiPA, servizi PA a Persone PA
+description: Il nuovo NoiPA costituirà il sistema unico di gestione del personale di tutta la Pubblica Amministrazione Italiana, che permetterà di accrescere la quantità e la tipologia dei servizi erogati dall’attuale applicazione, innovandone il modello di erogazione e favorendo un percorso di trasformazione digitale che utilizzi i nuovi paradigmi del cloud e dei big data.
```
Comment (alranel) [inline _platforms/it/noipa.md]: Same as above, let's try to rephrase it thinking about developers as our target

**Pull request merged**